src/DropDown/DropDown.component.tsx

```diff
@@ -60,48 +81,122 @@ const StyledDropDownContainer = styled(DropDownContainer)<
 >`
     min-width: 10rem;
     position: absolute;
-    padding: 0.25rem 0;
+    padding: 0.25rem auto;
     z-index: 1;
     background-color: ${th.color('neutrals.white.base')};
-    box-shadow: 0 0.5rem 0.5rem 0 ${th.color('neutrals.ash.base')};
+    box-shadow: 0 0 0.5rem 0 ${th.color('neutrals.ash.base')};
     border-radius: base;
     border: thin solid ${th.color('neutrals.silver.dark')};
+
     /* Defined Position */
-    top: ${({ position: { top } = {} }) =>
-        top ? `${top}px !important` : 'inherit'};
-    left: ${({ position: { left } = {} }) =>
-        left ? `${left}px !important` : '0'};
-    right: ${({ position: { right } = {} }) =>
-        right ? `${right}px !important` : 'inherit'};
-    bottom: ${({ position: { bottom } = {} }) =>
-        bottom ? `${bottom}px !important` : 'inherit'};
+    ${({ position, height, width, offset = '0.5rem' }) => {
```

Comment:
`offset` doesn't appear to be used

Reply:
See line 20. This isn't something I wanted to ship with MVP, but this will likely be an optimization we need.

Reply:
Got it.